flink-table/flink-table-planner/src/test/resources/restore-tests/stream-exec-ml-predict-table-function_1/sync-ml-predict-with-runtime-options/plan/sync-ml-predict-with-runtime-options.json

```diff
@@ -0,0 +1,137 @@
+{
+  "flinkVersion" : "2.1",
+  "nodes" : [ {
+    "id" : 10,
+    "type" : "stream-exec-table-source-scan_1",
+    "scanTableSource" : {
+      "table" : {
+        "identifier" : "`default_catalog`.`default_database`.`features`",
+        "resolvedTable" : {
+          "schema" : {
+            "columns" : [ {
+              "name" : "id",
+              "dataType" : "INT NOT NULL"
+            }, {
+              "name" : "feature",
+              "dataType" : "VARCHAR(2147483647)"
+            } ],
+            "watermarkSpecs" : [ ],
+            "primaryKey" : {
+              "name" : "PK_id",
+              "type" : "PRIMARY_KEY",
+              "columns" : [ "id" ]
+            }
+          },
+          "partitionKeys" : [ ]
+        }
+      }
+    },
+    "outputType" : "ROW<`id` INT NOT NULL, `feature` VARCHAR(2147483647)>",
+    "description" : "TableSourceScan(table=[[default_catalog, default_database, features]], fields=[id, feature])",
+    "inputProperties" : [ ]
+  }, {
+    "id" : 11,
+    "type" : "stream-exec-ml-predict-table-function_1",
+    "mlPredictSpec" : {
+      "features" : [ {
+        "type" : "FieldRef",
+        "index" : 1
+      } ],
+      "runtimeConfig" : {
+        "async" : "false"
+      }
+    },
+    "modelSpec" : {
+      "model" : {
+        "identifier" : "`default_catalog`.`default_database`.`chatgpt`",
+        "resolvedModel" : {
+          "inputSchema" : {
+            "columns" : [ {
+              "name" : "feature",
+              "dataType" : "VARCHAR(2147483647)"
+            } ],
+            "watermarkSpecs" : [ ]
+          },
+          "outputSchema" : {
+            "columns" : [ {
+              "name" : "category",
+              "dataType" : "VARCHAR(2147483647)"
+            } ],
+            "watermarkSpecs" : [ ]
```

Comment:
We don't need `watermarkSpecs` for model?

Reply:
ResolvedInputSchema and ResolvedOutputSchema both are instance of ResolvedSchema, and the ResolvedSchema seralizer and deserializer will serialize or deserialize this by default. If we want to remove this field, I perfer to align the concept first. WDYT?